cli.js

```diff
@@ -93,11 +93,20 @@ if (cli.flags.init) {
 	return;
 }
 
+var additionalPaths = [];
+if (process.env.NODE_PATH) {
+	var osSplitChar = process.platform === 'win32' ? ';' : ':';
+	process.env.NODE_PATH.split(osSplitChar).forEach(function (additionalPath) {
+		additionalPaths.push(path.resolve(opts.projectRoot, additionalPath));
```

Comment:
`process.cwd()` rather than `opts.projectRoot` right?

Perhaps `nodePaths` as the variable / option name? And personally I'd do something like:

```js
var nodePaths
if (process.env.NODE_PATH) {
	var osSplitChar = process.platform === 'win32' ? ';' : ':';
	nodePaths = process.env.NODE_PATH.split(osSplitChar).map(function (p) {
		return path.resolve(process.cwd(), p)
	});
} else {
	nodePaths = []
}
```